src/components/PopupWithConfirm.js

```diff
@@ -0,0 +1,29 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithConfirm extends Popup {
+  constructor(popupSelector) {
+    super({ popupSelector });
+    this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._confirmButton = document.querySelector("#delete-confirm");
+    this._saveButton = document.querySelector(".modal__button");
+  }
+
+  _setEventListeners() {
+    super._setEventListeners();
+    this._popupForm.addEventListener("submit", (e) => {
+      e.preventDefault();
+      this._handleDeleteSubmit();
+    });
+  }
```

Comment:
* When I delete 3 cards in a row there are caught errors about `remove`. See here http://joxi.ru/5md94LMSJnYwJ2 


There is a memory leak because the parend class `Popup` calls `_setEventListeners` in `open` (so, every time you open the modal)